libs/sdk-core/src/breez_services.rs

```diff
@@ -499,6 +500,35 @@ impl BreezServices {
         get_lsp_by_id(self.persister.clone(), self.lsp_api.clone(), id.as_str()).await
     }
 
+    pub async fn open_channel_fee(
+        &self,
+        req: OpenChannelFeeRequest,
+    ) -> SdkResult<OpenChannelFeeResponse> {
+        let lsp_info = self.lsp_info().await?;
+        let used_fee_params =
+            lsp_info.cheapest_open_channel_fee(req.expiry.unwrap_or(INVOICE_PAYMENT_FEE_EXPIRY))?;
```

Comment:
This could error also if there is no channel open needed.

Reply:
That's right, good catch!
I see that I will also need to change the `used_fee_params` to optional on the `OpenChannelFeeResponse`
For this specific case you mention here.

Reply:
Done